geomesa-hbase/pom.xml

```diff
@@ -32,6 +32,11 @@
                 <artifactId>geomesa-hbase-datastore_${scala.binary.version}</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.google.guava</groupId>
+                <artifactId>guava</artifactId>
+                <version>${hbase.guava.version}</version>
```

Comment:
I'm not sure, need @jnh5y to chime in. It was added as part of https://github.com/ccri/geomesa/commit/b9083ee473e3659e42db2a77b190c8640ac2dcff to get the density stuff to work but I'm not sure how to test it at this moment (need merges).